src/core/store/reducers/captures.js

```diff
@@ -6,7 +6,7 @@ const initialState = {}
 
 const stateKey = arr => cleanFalsy(arr).join('_')
 const getKeyByCaptureId = (captures = {}, captureId) =>
-  Object.keys(captures).find(key => captures[key].id === captureId)
+  Array.find(Object.keys(captures), key => captures[key].id === captureId)
```

Comment:
Is there are an eslint rule to disable certain method calls?

Reply:
No 😢 It looks like we'd have to write our own rule if we wanted this in linting.
Tbh I'm surprised that it's a thing that no-one else needs... 😞